.github/workflows/tests.yml

```diff
@@ -0,0 +1,366 @@
+name: Tests
+
+on:
+    push:
+        branches:
+            - 3.3.x
+            - master
+            - ticket/16659
```

Comment:
What is this for?

Reply:
Thats so he can show us the actions are actually working in the link he posted. Will be removed before merging.